test/DependencyInjection/Fixture/config/xml/event_bus.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+
+<srv:container xmlns="http://getprooph.org/schemas/symfony-dic/prooph"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xmlns:srv="http://symfony.com/schema/dic/services"
+               xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                        http://getprooph.org/schemas/symfony-dic/prooph http://getprooph.org/schemas/symfony-dic/prooph/service_bus-5.1.xsd">
+    <config>
+        <event_bus name="main" message_factory="Prooph\Common\Messaging\MessageFactory">
+            <plugin>Prooph\ServiceBus\Plugin\InvokeStrategy\OnEventStrategy</plugin>
+            <router type="Prooph\ServiceBus\Plugin\Router\EventRouter">
+                <route event="Prooph\ProophessorDo\Model\Todo\Event\TodoWasPosted">
+                    <value>Prooph\ProophessorDo\Projection\Todo\TodoProjector</value>
+                    <value>Prooph\ProophessorDo\Projection\Todo\UserProjector</value>
```

Comment:
@codeliner listener sounds good. Your XML example is also possible, but I want to avoid repeating the event name.